src/unitxt/metrics.py

```diff
@@ -111,21 +110,15 @@ def set_global_score(instances, global_score: Dict[str, Any]):
                 scores["global"] = {}
```

Comment:
this whole if sentence seems redundant to me. In any case, the following assignment sentence sets the final value of `scores["global"]`

Reply:
i agree, i think the `if` can be removed.. i can't think of why i added it..